docker/install/ubuntu_install_vitis_ai_packages_ci.sh

```diff
@@ -23,7 +23,7 @@ set -o pipefail
 export PYXIR_HOME=/opt/pyxir
 mkdir "$PYXIR_HOME"
 
-pip3 install progressbar
+pip3 install progressbar h5py==2.10.0
 
-git clone --recursive --branch v0.1.3 https://github.com/Xilinx/pyxir.git "${PYXIR_HOME}"
+git clone --recursive --branch v0.1.6 https://github.com/Xilinx/pyxir.git "${PYXIR_HOME}"
```

Comment:
Do you think it worth adding a `--depth 1` here so that you only clone the last commit in the branch?

Reply:
Yes, I added it.